nre/sui_for_node_operators.md

```diff
@@ -0,0 +1,292 @@
+```
+
+See the [full list of metadata update functions here](https://github.com/MystenLabs/sui/blob/main/crates/sui-framework/sources/governance/sui_system.move#L413-L553).
+
+### Operation Cap
+
+To avoid touching account keys too often and allowing them to be stored off-line, validators can delegate the operation ability to another address. This address can then update the reference gas price and tallying rule on behalf of the validator.
+
+Upon creating a `Validator`, an `UnverifiedValidatorOperationCap` is created as well and transferred to the validator address. The holder of this `Cap` object (short for "Capability") therefore could perform operational actions for this validator. To authorize another address to conduct these operations, a validator transfers the object to another address that they control. The transfer can be done by using Sui Client CLI: `sui client transfer`.
+
+To rotate the delegatee address or revoke the authorization, the current holder of `Cap` transfers it to another address. In the event of compromised or lost keys, the validator could create a new `Cap` object to invalidate the incumbent one. This is done by calling `sui_system::rotate_operation_cap`:
+
+```
+sui client call --package 0x2 --module sui_system --function rotate_operation_cap --args 0x5 --gas-budget 10000
+```
+
+By default the new `Cap` object is transferred to the validator address, which then could be transferred to the new delegatee address. At this point, the old `Cap` becomes invalidated and no longer represents eligibility.
+
+To get the current valid `Cap` object's ID of a validator, use the Sui Client CLI `sui client objects` command after setting the holder as the active address. Or go to the [explorer](https://explorer.sui.io/object/0x0000000000000000000000000000000000000005) and look for `operation_cap_id` of that validator in the `validators` module.
+
+### Updating the Reference Gas Price
+
+To update the Reference Gas Price (RGP), the sender needs to hold a valid [`UnverifiedValidatorOperationCap`](#operation-cap). The sender could be the validator itself, or a trusted delegatee. To do so, call `sui_system::request_set_gas_price`:
+
+```
+sui client call --package 0x2 --module sui_system --function request_set_gas_price --args 0x5 {cap_object_id} {new_gas_price} --gas-budget 10000
+```
+
+### Reporting/Un-reporting Validators
+
+To report a validator or undo an existing reporting, the sender needs to hold a valid [`UnverifiedValidatorOperationCap`](#operation-cap). The sender could be the validator itself, or a trusted delegatee. To do so, call `sui_system::report_validator/undo_report_validator`:
+
+```
+sui client call --package 0x2 --module sui_system --function report_validator/undo_report_validator --args 0x5 {cap_object_id} {reportee_address} --gas-budget 10000
+```
+
+### Joining or Leaving the Validator Set
+
+- TODO: get content from Emma
```

Comment:
Hmm the commit I added earlier seems to have disappeared? https://github.com/MystenLabs/sui/commit/b56a6f45c0718f5d49b951771c163899d58e594c

Reply:
ill add it - thanks